bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/IRobotBindingConstants.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.irobot.internal;
+
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link VacuumBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author hkuhn42 - Initial contribution
+ */
+public class IRobotBindingConstants {
+
+    public static final String BINDING_ID = "irobot";
+
+    // List of all Thing Type UIDs
+    public final static ThingTypeUID THING_TYPE_ROOMBA = new ThingTypeUID(BINDING_ID, "roomba");
```

Comment:
The SAT check complains about the order of the modifiers.
```suggestion
    public static final ThingTypeUID THING_TYPE_ROOMBA = new ThingTypeUID(BINDING_ID, "roomba");
```